apps/mull-ui/src/app/components/event-card/event-card.tsx

```diff
@@ -0,0 +1,58 @@
+import { IEvent } from '@mull/types';
+import React, { useState } from 'react';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faShareAlt, faCheck, faSignInAlt } from '@fortawesome/free-solid-svg-icons';
+
+import { formatDate } from '../../../utilities';
+
+import './event-card.scss';
+
+/* eslint-disable-next-line */
+export interface EventCardProps {
+  event: Partial<IEvent>;
+  style?: React.CSSProperties;
+}
+
+export const EventCard = ({ event, style = {} }: EventCardProps) => {
+  // TODO; set joined based on if current user is part of event
+  const [joined, setJoined] = useState<boolean>(false);
+
+  const { day, month, time } = formatDate(event.startDate);
+  return (
+    <div className="event-card-container" style={style}>
+      <img
+        className="event-card-image"
+        // TODO: Replace placeholder
+        src="https://www.citywindsor.ca/residents/parksandforestry/City-Parks/PublishingImages/Assumption%20Park%20Street%20View.JPG"
+        alt="Event"
+      />
+      <div className="event-card-datetime" data-testid="event-card-datetime">
+        <div>{`${month} ${day}`}</div>
+        <div>{time}</div>
+      </div>
+      {/* TODO: Add/remove user to event on press */}
+      <button
+        onClick={() => setJoined(!joined)}
+        className={`event-card-join ${joined ? 'joined' : ''}`}
+      >
+        <FontAwesomeIcon icon={joined ? faCheck : faSignInAlt} />
+      </button>
+      <div className="event-card-description">
+        <div className="event-card-text">
+          <div className="event-card-title">{event.title}</div>
+          {/* TODO: Replace placeholder distance*/}
+          <div className="event-card-location">14km • {event.location.point}</div>
+        </div>
+        {/* TODO: Replace placeholder */}
+        <div className="event-card-friends">friends</div>
```

Comment:
How come the styling for the friends part of the card is not implemented?

Reply:
I feel like that would be a seperate component, and its used in a few places in the application. I could create a component for it, but the images would all be placeholders

Reply:
I only saw it in event-card if it's in other places it could be a separate task. If not we'd want it to be done in this

Reply:
The event page is also meant to have it, currently, the icon that's there was more meant as a placeholder
![image](https://user-images.githubusercontent.com/23544999/100489965-ba873800-30e5-11eb-85ec-883a625318c3.png)

Reply:
oh okay yeah let's have it as a separate task then, I will make it now

Reply:
thanks ragith